src/Installers/Windows/GenerateNugetPackageWithMsi.ps1

```diff
@@ -25,9 +25,8 @@ if (-not (Test-Path $NuGetDir)) {
 }
 
 if (-not (Test-Path $NuGetExe)) {
-    # Using 5.3.0 to workaround https://github.com/NuGet/Home/issues/5016
```

Comment:
@rrelyea this bug doesn't appear relevant anymore, especially because we aren't using the more-performant 3.5.0 version. If that's correct, I'm wondering if is `dotnet pack` able to handle https://github.com/dotnet/aspnetcore/blob/master/src/Installers/Windows/SharedFramework/SharedFrameworkPackage.nuspec and https://github.com/dotnet/aspnetcore/blob/master/src/Installers/Windows/TargetingPack/TargetingPackPackage.nuspec as well as the properties we pass in below yet ❔ I don't see `*.nuspec`support in the help `dotnet pack -?` spits out but hope it's just undocumented 😄 or will be coming soon.

Reply:
Chatted offline w/ @rrelyea and it's likely we could create the two installer packages using a new `*.csproj` that runs after our `*.wixproj`.

Will file a separate issue about this work. It's a larger change and not something I'm going to tackle now. But, it would remove our need for NuGet.exe anywhere in this repo (outside a couple of Arcade post-build stages).